packages/grafana-data/src/types/config.ts

```diff
@@ -85,6 +86,43 @@ export type OAuth =
  */
 export type OAuthSettings = Partial<Record<OAuth, { name: string; icon?: string }>>;
 
+/** Current user info included in bootData
+ *
+ * @public
+ */
+export interface CurrentUserDTO {
```

Comment:
Any thoughts on which of these should/shouldn't be public? I've moved `helpFlags1` and `hasEditPermissionsInFolder` to a separate interface that extends this one, but perhaps there are others? @torkelo @mckn